galleries.py

```
@@ -512,376 +534,177 @@ def _gallery(fileroot, product_ids, catalog, links,
+                previous_link = url
+ 
+                # jump to navigation
+                yaml_data['jump_to'] = create_jump_to_links(links)
+
+            # image blocks
+            yaml_data['image_table'] = create_floated_img_blocks(url)
+
+            write_yaml_file(url, yaml_data)
+   
+        else:
+            for subindex, subitem in enumerate(item):
```

Ruff (0.14.14)

[warning] Loop control variable `subindex` not used within loop body

(B007)